x/adminmodule/types/keys.go

```diff
@@ -23,12 +23,17 @@ const (
 	AdminKey = "Admin-"
 
 	ArchiveKey = "Archive-"
+
+	ArchiveLegacyKey = "ArchiveLegacy-"
```

Comment:
seems like state breaking change. I mean, do we need to move all ArchivedProposals from `Archive-` to `ArchiveLegacy-` and clear `Archive-`

Reply:
No. As soon as legacy proposals has the same key & and structure as before, we change only keys in terms of golang variables